plugins/stk/mallets/mallets.cpp

```diff
@@ -620,7 +620,12 @@ malletsSynth::malletsSynth( const StkFloat _pitch,
 	{
 		Stk::setSampleRate( _sample_rate );
 		Stk::setRawwavePath( QDir( ConfigManager::inst()->stkDir() ).absolutePath()
-						.toLatin1().constData() );
+#ifdef LMMS_BUILD_WIN32
+						.toLocal8Bit().constData() );
+#else
+						.toUtf8().constData() );
+#endif
+
```

Comment:
I think we can (should?) use `toLocal8Bit` on Unix as well. It should be equivalent to `toUtf8` on most Linux and macOS systems. Correct me if I'm wrong.